scripts/2_munge/sankey_fig.R

```diff
@@ -0,0 +1,227 @@
+fish.change = read.csv('data/categories_by_time_period_median_probs.csv', stringsAsFactors = FALSE)
```

Comment:
this file should not exist anymore - it has been moved into processFutureSuitability and visualizeFutureSuitability